release_docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,159 @@
+        - `$ git push` 
+    7. Update default configuration mode
+        - `$ git checkout hdf5_X_Y_Z;` and `$ bin/switch_maint_mode -disable ./configure.ac` to disable `AM_MAINTAINER_MODE`. 
+        - Need to set option `HDF5_GENERATE_HEADERS` to `OFF`, currently in line 996 of [src/CMakeLists.txt](https://github.com/HDFGroup/hdf5/blob/develop/src/CMakeLists.txt).
+        - Change the **release preparation branch**'s (i.e. hdf5_X_Y_Z) default configuration mode from development to production in [configure.ac](https://github.com/HDFGroup/hdf5/blob/develop/configure.ac). 
+        - Find “Determine build mode” in [configure.ac](https://github.com/HDFGroup/hdf5/blob/develop/configure.ac). 
+        - Change `default=debug` to `default=production` at the bottom of the `AS_HELP_STRING` for `--enable-build-mode`.
+        - Under `if test "X-$BUILD_MODE" = X- ; then` change `BUILD_MODE=debug` to `BUILD_MODE=production`. 
+        - Run `“sh ./autogen.sh”` to regenerate the UNIX build system files and commit the changes. (use `git status --ignored` to see the changes and `git add -f` to add all files. First delete any new files not to be committed, notably `src/H5public.h~` and `autom4te.cache/`.)
+        - `$ git push with commit message listing change steps for creating release branch`
+    8. E-mail hdf5lib@hdfgroup.org to indicate that the code freeze on the release support branch (i.e. hdf5_X_Y) has been lifted and development on the next maintenance release can resume. The code freeze will remain in place on the release preparation branch (i.e. hdf5_X_Y_Z) indefinitely. 
+
+7. Perform Release Testing (Test Automation Team | Release Manager | Project Leads)
+    1. Verify that source and binary distributions of HDF5 are acceptable on all target operating environments.
+    2. Create a page on Confluence as a sub-page of the current release version's project collaboration page (see HDF5 Maintenance Releases) to document release testing results. 
+    3. Document the test procedure that will be used for this release on the new sub-page. 
+    4. Schedule daily stand-up meetings or other regular checkpoints to assure progress on testing tasks, and inform testing team of your expectations.
+    5. Schedule and enable daily automated regression testing of the release preparation branch (i.e. hdf5_X_Y_Z). Give the 'OK' to proceed once all required tests have verified that HDF5 is functioning as intended on all target operating environments. 
+    6. Select release build from workflow.
+    7. Choose the release branch
+    8. Change ‘Release version tag’ name to 'hdf5_X.Y.Z.P'
+        - P is some pre-release number.
+    9. Send a message to the HDF forum indicating that a pre-release source package is available for testing at <e.g. gamma.hdfgroup.org/ftp/pub/outgoing/hdf5/{hdf5-X.Y.Z-pre<n>}> and that feedback from the user community on their test results is being accepted.
```

Comment:
'P' vs "pre\<n\>"?

Reply:
Hi Binh-Minh, I tried updating this link, but it didn't seem to pass the links check once I did, so I have reverted the change.